README.md

```diff
@@ -31,79 +31,68 @@ There is a set of samples in the `samples` folder to help in getting started wit
-
-    - App-only access via Client Credential with a certificate.
-
-        The certificate will be loaded from `Cert:\CurrentUser\My\` store when `-CertificateThumbprint` or `-CertificateName` is specified. Ensure the certificate is present in the store before calling `Connect-MgGraph`.
+    ``` powershell
+    # Using interactive authentication.
+    Connect-MgGraph -Scopes "User.Read.All", "Group.ReadWrite.All"
+    ```
 
-        ``` powershell
-        # Using -CertificateThumbprint
-        Connect-MgGraph -ClientId "YOUR_APP_ID" -TenantId "YOUR_TENANT_ID" -CertificateThumbprint "YOUR_CERT_THUMBPRINT"
-        ```
+    or
 
-        or
+    ``` powershell
+    # Using device code flow.
+    Connect-MgGraph -Scopes "User.Read.All", "Group.ReadWrite.All" -UseDeviceAuthentication
+    ```
 
-        ``` powershell
-        # Using -CertificateName
-        Connect-MgGraph -ClientId "YOUR_APP_ID" -TenantId "YOUR_TENANT_ID" -CertificateName "YOUR_CERT_SUBJECT"
-        ```
+    or
 
-        or
+    ``` powershell
+    # Using your own access token.
+    Connect-MgGraph -AccessToken $AccessToken
+    ```
 
-        ``` powershell
-        # Using -Certificate
-        $Cert = Get-ChildItem Cert:\LocalMachine\My\$CertThumbprint
-        Connect-MgGraph -ClientId "YOUR_APP_ID" -TenantId "YOUR_TENANT_ID" -Certificate $Cert
-        ```
+- App-only access via Client Credential with a certificate.
 
-2. List users in your tenant.
+    The certificate will be loaded from `Cert:\CurrentUser\My\` store when `-CertificateThumbprint` or `-CertificateName` is specified. Ensure the certificate is present in the store before calling `Connect-MgGraph`.
 
     ``` powershell
-    Get-MgUser -Top 10 -Property Id, DisplayName, BusinessPhones | Format-Table Id, DisplayName, BusinessPhones
+    # Using -CertificateThumbprint
+    Connect-MgGraph -ClientId "YOUR_APP_ID" -TenantId "YOUR_TENANT_ID" -CertificateThumbprint "YOUR_CERT_THUMBPRINT"
     ```
 
-3. Filter a user in your tenant.
+    or
 
     ``` powershell
-    $user = Get-MgUser -Filter "displayName eq 'Megan Bowen'"
+    # Using -CertificateName
+    Connect-MgGraph -ClientId "YOUR_APP_ID" -TenantId "YOUR_TENANT_ID" -CertificateName "YOUR_CERT_SUBJECT"
     ```
 
-4. Create a new app registration.
+    or
 
     ``` powershell
-    New-MgApplication -DisplayName "ScriptedGraphPSApp" `
-                      -SignInAudience "AzureADMyOrg" `
-                      -Web @{ RedirectUris = "https://localhost"}
+    # Using -Certificate
+    $Cert = Get-ChildItem Cert:\LocalMachine\My\$CertThumbprint
+    Connect-MgGraph -ClientId "YOUR_APP_ID" -TenantId "YOUR_TENANT_ID" -Certificate $Cert
     ```
 
-5. Sign out of the current logged-in context i.e. app only or delegated access.
+1. Sign out of the current logged-in context i.e. app only or delegated access.
 
     ``` powershell
     Disconnect-MgGraph
     ```
+## More usage
+
+* [Applications](./CmdletsDescription/Applications/Application.md)
```

Comment:
Let's move these to `docs` directory instead of `CmdletsDescription` to be consistent with other SDKs.